frontend/src/app/components/fileDeleteRespons.ts

```diff
@@ -0,0 +1,12 @@
+export class FileDeleteRespons {
```

Comment:
That task is in a different branch not yet merged, hence there might be some changes to them. Shouldn't add them in this branch.